concrete/blocks/page_list/controller.php

```diff
@@ -798,6 +806,7 @@ public function save($args)
         $args['excludeCurrentPage'] = ($args['excludeCurrentPage']) ? '1' : '0';
         $args['truncateChars'] = (int) ($args['truncateChars']);
         $args['paginate'] = (int) ($args['paginate']);
+        $args['excludeCanonicalPaging'] = (int) ($args['excludeCanonicalPaging']);
```

Comment:
If `$args` doesn't contain the `'excludeCanonicalPaging'` we'll have a warning.

What about replacing this line with

```php
$args['excludeCanonicalPaging'] = (int) ($args['excludeCanonicalPaging'] ?? 0);
```

Reply:
@mlocati Thank you, I'll make note of this!

At the moment I'm having issues with the migration failing to match version numbers.  My semi educated guess is it's something to do with the base branch switching to 9.4.x and my fork being stuck on pre 9.4.x updates ?!  I'm unable to update my branch from concrete with the modifications for this PR intact.  Unless someone has some command line ninja stuff I can apply I may just start this over :|